LogMonitor/LogMonitorTests/LogFileMonitorTests.cpp

```diff
@@ -673,12 +671,11 @@ namespace LogMonitorTests
             sourceFile.Directory = tempDirectory;
             sourceFile.Filter = L"*.log";
             sourceFile.IncludeSubdirectories = true;
-            sourceFile.IncludeFileNames = true;
 
             fflush(stdout);
             ZeroMemory(bigOutBuf, sizeof(bigOutBuf));
 
-            std::shared_ptr<LogFileMonitor> logfileMon = std::make_shared<LogFileMonitor>(sourceFile.Directory, sourceFile.Filter, sourceFile.IncludeSubdirectories, sourceFile.IncludeFileNames);
+            std::shared_ptr<LogFileMonitor> logfileMon = std::make_shared<LogFileMonitor>(sourceFile.Directory, sourceFile.Filter, sourceFile.IncludeSubdirectories);

```

Comment:
nit: the lines above a repeated too frequently, perhaps a function with default settings would make these changes a bit less repetitive. Can be address later

Reply:
Let me see what to do about this... I think the previous authors could have done like this since these are tests...